cirq/ops/pauli_string_test.py

```diff
@@ -1600,25 +1600,3 @@ def test_deprecated():
         _ = cirq.PauliString({
             a: 'x'
         }).expectation_from_state_vector(state=state_vector, qubit_map={a: 0})
-
-
-def test_circuit_diagram_info():
```

Comment:
We should bring back this test too after we brought back the diagram info for PauliStrings.